blocks/footer/footer.js

```diff
@@ -7,6 +7,36 @@ function addFooterInteractions(block) {
     window.scrollTo(0, 0);
   };
 
+  const accordionItems = block.querySelectorAll('.accordion__item');
+
+  if (accordionItems) {
+    accordionItems.forEach((ele) => {
+      const accordionButton = ele.querySelector('.accordion__button');
+      const accordionIconSymbol = accordionButton ? accordionButton.querySelector('.icon--symbol') : null;
+      const accordionButtonSibling = accordionButton ? accordionButton.nextElementSibling : null;
+
+      if (accordionButton && accordionButtonSibling) {
+        accordionButtonSibling.classList.toggle('display-hide');
+
+        accordionButton.onclick = () => {
+          const countryAccordionButtons = document.querySelectorAll('.footer-country-switch .accordion__button');
+
+          countryAccordionButtons.forEach((x) => {
```

Comment:
Probably we should add Null safety check for `countryAccordionButtons`
Similarly for `countryButtonSibling`